libdnf/base/Base.hpp

```diff
@@ -23,10 +26,14 @@ namespace libdnf {
 /// ime needed to accomplish its packaging tasks. Plugins are managed by DNF and get a reference to :class:`dnf.Base` object when they run.
 /// :class:`.Base` instances are stateful objects holding references to various data sources and data sinks. To properly finalize and close off any handles the object may hold, client code
 /// should either call :meth:`.Base.close` when it has finished operations with the instance, or use the instance as a context manager. After the object has left the context, or its :meth:`.Base.
+// lukash: Is this the python documentation talking about context manager? Lets translate that into C++
 /// close` has been called explicitly, it must not be used. :meth:`.Base.close` will delete all downloaded packages upon successful transaction.
```

Comment:
```
-
-/// Instances of :class:`dnf.Base` are the central point of functionality supplied by DNF. An application will typically create a single instance of this class which it will keep for the runt
-/// ime needed to accomplish its packaging tasks. Plugins are managed by DNF and get a reference to :class:`dnf.Base` object when they run.
-/// :class:`.Base` instances are stateful objects holding references to various data sources and data sinks. To properly finalize and close off any handles the object may hold, client code
-/// should either call :meth:`.Base.close` when it has finished operations with the instance, or use the instance as a context manager. After the object has left the context, or its :meth:`.Base.
-// lukash: Is this the python documentation talking about context manager? Lets translate that into C++
-/// close` has been called explicitly, it must not be used. :meth:`.Base.close` will delete all downloaded packages upon successful transaction.
+/// Instances of :class:`dnf.Base` are the central point of functionality supplied by DNF. An
+/// application will typically create a single instance of this class which it will keep for the
+/// runt ime needed to accomplish its packaging tasks. Plugins are managed by DNF and get a
+/// reference to :class:`dnf.Base` object when they run. :class:`.Base` instances are stateful
+/// objects holding references to various data sources and data sinks. To properly finalize and
+/// close off any handles the object may hold, client code should either call :meth:`.Base.close`
+/// when it has finished operations with the instance, or use the instance as a context manager.
+/// After the object has left the context, or its :meth:`.Base.
+// lukash: Is this the python documentation talking about context manager? Lets translate that into
+// C++
+/// close` has been called explicitly, it must not be used. :meth:`.Base.close` will delete all
+/// downloaded packages upon successful transaction.
```